x/ibc/20-transfer/module.go

```diff
@@ -119,3 +124,123 @@ func (am AppModule) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 func (am AppModule) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) []abci.ValidatorUpdate {
 	return []abci.ValidatorUpdate{}
 }
+
+// Implement IBCModule callbacks
+func (am AppModule) OnChanOpenInit(
+	ctx sdk.Context,
+	order channelexported.Order,
+	connectionHops []string,
+	portID string,
+	channelID string,
+	counterparty channeltypes.Counterparty,
+	version string,
+) error {
+	// TODO: Enforce ordering, currently relayers use ORDERED channels
```

Comment:
Should we enforce ORDERING to UNORDERED as mentioned in spec? Currently all channels doing transfers on relayer are ORDERED channels.

@cwgoes @jackzampolin

Reply:
In principle, token transfer channels can safely be unordered. I think we should test the relayer with both.